lib/mixins/id.js

```diff
@@ -24,5 +24,11 @@ export default {
         safeId() {
             return this.id || this.localId_ || null;
         }
+    },
+    methods: {
+        safeIdPlus(str) {
+            str = String(str || '').replace(/\s+/g, '_');
+            return (this.safeId && Boolean(str)) ? `${this.safeId}_${str}` : null;
+        }
     }
 };
```

Comment:
True, but in most cases, this is an internal ID's will have suffix only.

Originally I had planned on a single method, which appended the suffix if provided (similar to what you have above), but I wanted wanted the suffixed ID to return null if the suffix was invalid (i.e. a boolean, undefined, etc).